lib/datasource/cdnjs/index.spec.ts

```diff
@@ -1,12 +1,9 @@
 import { getPkgReleases } from '..';
+import { Fixtures } from '../../../test/fixtures';
 import * as httpMock from '../../../test/http-mock';
-import { loadFixture } from '../../../test/util';
 import { EXTERNAL_HOST_ERROR } from '../../constants/error-messages';
 import { CdnJsDatasource } from '.';
 
-const res1 = loadFixture('d3-force.json');
-const res2 = loadFixture('bulma.json');
```

Comment:
I think we should do this wherever possible to allow better garbage collection in tests and reduce memory pressure.

Reply:
I agree, applied similar changes for more places